### PR TITLE
Add Mobile Select Filters

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 7 errors:

* corehq/apps/app_manager/tests/test_report_config.py, line 46 - E501 line too long (135 > 115 characters)
* corehq/apps/app_manager/tests/test_report_config.py, line 107 - E501 line too long (389 > 115 characters)
* corehq/apps/app_manager/tests/test_report_config.py, line 108 - E501 line too long (331 > 115 characters)
* corehq/apps/app_manager/tests/test_report_config.py, line 133 - E501 line too long (128 > 115 characters)
* corehq/apps/app_manager/tests/test_report_config.py, line 138 - E501 line too long (267 > 115 characters)
* corehq/apps/app_manager/tests/test_report_config.py, line 165 - E501 line too long (295 > 115 characters)
* corehq/apps/app_manager/tests/test_report_config.py, line 174 - E501 line too long (135 > 115 characters)

**Comment by dimagimon** [comment]
There are 7 errors:

* corehq/apps/app_manager/tests/test_report_config.py, line 46 - E501 line too long (135 > 115 characters)
* corehq/apps/app_manager/tests/test_report_config.py, line 107 - E501 line too long (389 > 115 characters)
* corehq/apps/app_manager/tests/test_report_config.py, line 108 - E501 line too long (331 > 115 characters)
* corehq/apps/app_manager/tests/test_report_config.py, line 133 - E501 line too long (128 > 115 characters)
* corehq/apps/app_manager/tests/test_report_config.py, line 138 - E501 line too long (267 > 115 characters)
* corehq/apps/app_manager/tests/test_report_config.py, line 165 - E501 line too long (295 > 115 characters)
* corehq/apps/app_manager/tests/test_report_config.py, line 174 - E501 line too long (135 > 115 characters)

**Comment by czue** [comment]
only took a quick glance but looking good. feel free to ping me/others when you want this reviewed

**Comment by czue** [comment]
(er, maybe it's ready now - though I think someone with app manager context or @nickpell / @orangejenny would be better than me)

**Comment by snopoke** [comment]
:+1: from me

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -3238,7 +3238,7 @@ class ReportGraphConfig(DocumentSchema):
 
 
 class ReportAppFilter(DocumentSchema):
-    def get_filter_value(self):
+    def get_filter_value(self, user):
```
Comment (nickpell) [inline corehq/apps/app_manager/models.py]: :+1:

**Comment by dannyroberts** [comment]
Oops my bad. Didn't intend to merge; this probably won't break anything
existing, but the feature won't work end to end yet so anyone who tries
will be confused and disappointed. Probably fine to leave it in the code as
I'll be fixing up the rest this week.

**Pull request merged**